### ISSUE TITLE
Empty Page

### ISSUE THREAD

**Issue by jensfr1** [sbormann/ioBroker.iqontrol#51]
**Describe the bug**  
iqontrol page is empty for end-user but device list is filled. 

Actually I figured out why. I created a empty device without any role and used it as header (not at the end of the list) Once it is created in a view the whole view is empty. 

![image](https://user-images.githubusercontent.com/12272992/59665995-42100e00-91b4-11e9-9cb4-1cd565501ac1.png)

**Comment by sbormann** [comment]
Stimmt! Wird korrigiert. Danke!

--
Von meinem iPhone aus gesendet

Am 18.06.2019 um 10:32 schrieb jensfr1 <notifications@github.com<mailto:notifications@github.com>>:


Describe the bug
iqontrol page is empty for end-user but device list is filled.

Actually I figured out why. I created a empty device without any role and used it as header (not at the end of the list) Once it is created in a view the whole view is empty.

[image]<https://user-images.githubusercontent.com/12272992/59665995-42100e00-91b4-11e9-9cb4-1cd565501ac1.png>
